cmd/collector/app/handler/zipkin_receiver.go

```diff
@@ -69,6 +69,9 @@ func startZipkinReceiver(
 	}
 
 	consumerAdapter := newConsumerDelegate(logger, spanProcessor, tm)
+	// reset Zipkin spanFormat
+	consumerAdapter.batchConsumer.spanOptions.SpanFormat = processor.ZipkinSpanFormat
```

Comment:
can you add a test for this? `TestZipkinReceiver` submits zipkin spans but uses a `mockSpanProcessor`, we could potentially enhance it to validate that those spans are coming with the desired tag

Reply:
Done.
@yurishkuro PTAL  :)